generators/client/templates/vue/src/test/javascript/spec/app/admin/audits/audits.component.spec.ts.ejs

```diff
@@ -90,7 +90,15 @@ describe('Audits Component', () => {
   describe('OnInit', () => {
     it('Should call load all on init', async () => {
       // GIVEN
-      mockedAxios.get.mockReturnValue(Promise.resolve({ headers: {}, data: ['test'] }));
+      mockedAxios.get.mockReturnValue(Promise.resolve({ headers: {}, data: 
```

Comment:
This resulted in a console error already before (but the test did not fail so it went unnoticed)